packages/turf-polygon-smooth/index.js

```diff
@@ -0,0 +1,55 @@
+import { coordEach, geomEach } from '@turf/meta';
+import { featureCollection, polygon, isObject } from '@turf/helpers';
+
+/**
+ * Smooths a {@link Polygon}. Based on [Chaikin's algorithm](http://graphics.cs.ucdavis.edu/education/CAGDNotes/Chaikins-Algorithm/Chaikins-Algorithm.html).
+ * Warning: may create degenerate polygons.
+ *
+ * @name polygonSmooth
+ * @param {FeatureCollection<Polygon>} inputPolys to smooth
+ * @param {Object} [options] Optional parameters
```

Comment:
Optional, I've been been giving an empty Object as the default value `[options={}]`